tests/php/test_class_wc-connect-nux.php

```diff
@@ -7,56 +7,91 @@ public static function setupBeforeClass() {
-			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_DEV,
-			'tos_accepted'                           => true,
-			'should_display_after_cxn_banner'        => false,
-		) ), false );
-
-		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
-			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_DEV,
-			'tos_accepted'                           => false,
-			'should_display_after_cxn_banner'        =>  true,
-		) ), false );
-
-		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
-			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_DEV,
-			'tos_accepted'                           => true,
-			'should_display_after_cxn_banner'        =>  true,
-		) ), false );
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => false,
+					'can_accept_tos'                  => false,
+					'should_display_after_cxn_banner' => false,
+				)
+			), false
+		);
+
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => true,
+					'can_accept_tos'                  => false,
+					'should_display_after_cxn_banner' => false,
+				)
+			), false
+		);
+
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => false,
+					'can_accept_tos'                  => false,
+					'should_display_after_cxn_banner' => true,
+				)
+			), 'after_jetpack_connection'
+		);
+
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => true,
+					'can_accept_tos'                  => false,
+					'should_display_after_cxn_banner' => true,
+				)
+			), 'after_jetpack_connection'
+		);
+
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => false,
+					'can_accept_tos'                  => true,
+					'should_display_after_cxn_banner' => false,
+				)
+			), 'tos_only_banner'
+		);
 	}
 
 	public function test_get_banner_type_to_display_no_jp_cxn_without_tos_acceptance() {
 		// before going through connection
 		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
 			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_NOT_INSTALLED,
 			'tos_accepted'                           => false,
-			'should_display_after_cxn_banner'        =>  false,
+			'can_accept_tos'                         => false,
```

Comment:
In all of these cases, if `can_accept_tos` is false, shouldn't we not display any banner?